reana_server/complexity.py

```diff
@@ -89,8 +93,8 @@ def _get_number_of_jobs(self, step):
 
     def _get_memory_limit(self, step):
         """Get memory limit value."""
-        memory_limit = step.get(
-            "kubernetes_memory_limit", REANA_COMPLEXITY_JOBS_MEMORY_LIMIT
+        memory_limit = (
+            step.get("kubernetes_memory_limit") or REANA_COMPLEXITY_JOBS_MEMORY_LIMIT
```

Comment:
*question:* Why replacing `dict.get("key", <defaut_value>)` with `dict.get("key") or <default_value>`? It has the same behaviour and `.get(key, default)` is proper use of Python API.

Reply:
In this particular case, it doesn't have the same behavior as `kubernetes_memory_limit` might be `None` in the step spec so the `memory_limit` would remain `None` when we want it to be `REANA_COMPLEXITY_JOBS_MEMORY_LIMIT` in that case.